tests/test_cli_tools.py

```diff
@@ -0,0 +1,268 @@
+""" Tests for rendering CLI options and arguments """
+
+from collections import OrderedDict
+import pytest
+from ubiquerg import build_cli_extra, powerset, VersionInHelpParser, convert_value
+import argparse
+
+
+def pytest_generate_tests(metafunc):
+    """ Test case generation and parameterization for this module """
+    if "ordwrap" in metafunc.fixturenames:
+        metafunc.parametrize("ordwrap", [tuple, OrderedDict])
+
+
+def build_parser():
```

Comment:
why exactly is the looper CLI parser duplicated in here?

Reply:
Needed a test example for VersionInHelp parser methods. I just chose not to spend time creating a new parser and reused the looper one.

Reply:
got it -- might be nice to just include a comment to that effect -- 'this is just an example' and this particular functionality is meaningless, so that in the future we're not confused?

Reply:
That’s what I tried to communicate in line 18. I’ll  add one more sentence to make it less confusing.